ui/packages/tidb-dashboard-for-clinic-cloud/src/apps/Monitoring/metricsQueries.ts

```diff
@@ -48,615 +48,702 @@ function transformColorByExecTimeOverview(legendLabel: string) {
-        queries: [
-          {
-            query: '(time() - process_start_time_seconds{job="tidb"})',
-            name: '{instance}'
-          }
-        ],
-        nullValue: TransformNullValue.AS_ZERO,
-        unit: 's',
-        type: 'line'
-      },
-      {
-        title: 'TiDB CPU Usage',
-        queries: [
-          {
-            query: 'rate(process_cpu_seconds_total{job="tidb"}[30s])',
-            name: '{instance}'
-          }
-        ],
-        nullValue: TransformNullValue.AS_ZERO,
-        unit: 'percentunit',
-        type: 'line'
-      },
-      {
-        title: 'TiDB Memory Usage',
-        queries: [
-          {
-            query: 'process_resident_memory_bytes{job="tidb"}',
-            name: '{instance}'
-          }
-        ],
-        nullValue: TransformNullValue.AS_ZERO,
-        unit: 'bytes',
-        type: 'line'
-      },
-      {
-        title: 'TiKV Uptime',
-        queries: [
-          {
-            query: '(time() - process_start_time_seconds{job="tikv"})',
-            name: '{instance}'
-          }
-        ],
-        unit: 's',
-        type: 'line'
-      },
-      {
-        title: 'TiKV CPU Usage',
-        queries: [
-          {
-            query:
-              'sum(rate(tikv_thread_cpu_seconds_total[$__rate_interval])) by (instance)',
-            name: '{instance}'
-          }
-        ],
-        unit: 'percentunit',
-        type: 'line'
-      },
-      {
-        title: 'TiKV Memory Usage',
-        queries: [
-          {
-            query: 'process_resident_memory_bytes{job=~".*tikv"}',
-            name: '{instance}'
-          }
-        ],
-        unit: 'bytes',
-        type: 'line'
-      },
-      {
-        title: 'TiKV IO MBps',
-        queries: [
-          {
-            query:
-              'sum(rate(tikv_engine_flow_bytes{db="kv", type="wal_file_bytes"}[$__rate_interval])) by (instance) + sum(rate(tikv_engine_flow_bytes{db="raft", type="wal_file_bytes"}[$__rate_interval])) by (instance) + sum(rate(raft_engine_write_size_sum[$__rate_interval])) by (instance)',
-            name: '{instance}-write'
-          },
-          {
-            query:
-              'sum(rate(tikv_engine_flow_bytes{db="kv", type=~"bytes_read|iter_bytes_read"}[$__rate_interval])) by (instance)',
-            name: '{instance}-read'
-          }
-        ],
-        unit: 'Bps',
-        type: 'line'
-      },
-      {
-        title: 'TiKV Storage Usage',
-        queries: [
-          {
-            query: 'sum(tikv_store_size_bytes{type="used"}) by (instance)',
-            name: '{instance}'
-          }
-        ],
-        unit: 'decbytes',
-        type: 'area_stack'
-      }
-    ]
+const getMonitoringItems = (
+  pdVersion: string | undefined
+): MetricsQueryType[] => {
+  function loadTiKVStoragePromql() {
+    const PDVersion = pdVersion?.replace('v', '')
+    console.log('PDVersion', PDVersion)
+
+    if (PDVersion && PDVersion < '5.4.1') {
```

Comment:
It May be better to use semver lib to compare the versions to avoid potential bugs, for example: `5.11.1 < 5.4.1`.